src/gui/entry/EditEntryWidget.cpp

```diff
@@ -763,20 +757,43 @@ void EditEntryWidget::updateEntryData(Entry* entry) const
 
     if (iconStruct.number < 0) {
         entry->setIcon(Entry::DefaultIconNumber);
-    }
-    else if (iconStruct.uuid.isNull()) {
+    } else if (iconStruct.uuid.isNull()) {
         entry->setIcon(iconStruct.number);
-    }
-    else {
+    } else {
         entry->setIcon(iconStruct.uuid);
     }
 
     entry->setAutoTypeEnabled(m_autoTypeUi->enableButton->isChecked());
     if (m_autoTypeUi->inheritSequenceButton->isChecked()) {
         entry->setDefaultAutoTypeSequence(QString());
-    }
-    else {
-        entry->setDefaultAutoTypeSequence(m_autoTypeUi->sequenceEdit->text());
+    } else {
```

Comment:
The whole else block is completely redundant. Please consolidate to avoid code duplication.